test/msw-api/cli/init.test.ts

```diff
@@ -133,3 +133,37 @@ test('warns when current public directory does not match the saved directory fro
 
   await cleanup()
 })
+
+test('does not cause ESLint errors or warnings', async () => {
+  const { prepare, getPath, cleanup } = createTeardown(
+    './tmp/cli/init/eslint',
+    addFile('package.json', {
+      name: 'package-name',
+    }),
+    // intentionally degenerate config to cause a warning in the initial comment
```

Comment:
"intentionally degenerate" love this 😆

Reply:
oh... actually it turns out "degenerate" doesn't mean exactly what I thought it did, heh :) "antagonistic" and/or "contrived" is probably more what I'm looking for

Reply:
All words are fine!